mlir/lib/Dialect/GPU/IR/InferIntRangeInterfaceImpls.cpp

```diff
@@ -95,6 +100,12 @@ void ClusterIdOp::inferResultRanges(ArrayRef<ConstantIntRanges>,
   setResultRange(getResult(), getIndexRange(0, max - 1ULL));
 }
 
+void ClusterBlockIdOp::inferResultRanges(ArrayRef<ConstantIntRanges>,
+                                         SetIntRangeFn setResultRange) {
+  uint64_t max = kMaxClusterDim;
+  setResultRange(getResult(), getIndexRange(0, max - 1ULL));
```

Comment:
IMHO, Let's not add this. What do you think?

Because non_portable_cluster_size attribute could be set on the kernel launch, but this verifier won't let you use more than 8 cluster size.

Reply:
Right, I was planning to update it in the subsequent PR when I introduced the non_portable_cluster_size attribute. I can update it in the PR as well

Reply:
sounds good